examples/query_params/src/main.rs

```diff
@@ -3,6 +3,8 @@
 
 extern crate rocket;
 
+#[cfg(test)] mod tests;
```

Comment:
Can we split these into two lines? I'm okay with this, actually, but it's just what the rest of the example files do.